src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,19 @@
+package racingcar;
+
+public class Car {
+
+    public static final int MOVE_CONDITION = 4;
+    private int currentPosition;
```

Comment:
상수와 인스턴스 변수의 구분을 위해 개행을 추가해보면 어떨까요?
```suggestion
    public static final int MOVE_CONDITION = 4;

    private int currentPosition;
```